gym/vector/async_vector_env.py

```diff
@@ -463,13 +562,29 @@ def _worker_shared_memory(index, env_fn, pipe, parent_pipe, shared_memory, error
             elif command == "close":
                 pipe.send((None, True))
                 break
+            elif command == "_call":
+                name, args, kwargs = data
+                if name in ["reset", "step", "seed", "close"]:
+                    raise ValueError(
+                        "Trying to call function `{0}` with "
+                        "`_call`. Use `{0}` directly instead.".format(name)
+                    )
+                function = getattr(env, name)
+                if callable(function):
+                    pipe.send((function(*args, **kwargs), True))
+                else:
+                    pipe.send((function, True))
+            elif command == "_setattr":
+                name, value = data
+                setattr(env, name, value)
+                pipe.send((None, True))
             elif command == "_check_observation_space":
                 pipe.send((data == observation_space, True))
             else:
                 raise RuntimeError(
                     "Received unknown command `{0}`. Must "
-                    "be one of {`reset`, `step`, `seed`, `close`, "
-                    "`_check_observation_space`}.".format(command)
+                    "be one of {`reset`, `step`, `seed`, `close`, `_call`, "
+                    "`_setattr`, `_check_observation_space`}.".format(command)
```

Comment:
I can't suggest a format string here, since the relevant line was only partially modified, but I'd probably use it here, too.